modules/rgbd/include/opencv2/rgbd/kinfu.hpp

```diff
@@ -16,6 +16,65 @@ namespace kinfu {
 //! @addtogroup kinect_fusion
 //! @{
 
+struct CV_EXPORTS_W VolumeParams
+{
+    enum VolumeKind
+    {
+        TSDF = 0,
+        HASHTSDF = 1,
+        COLOREDTSDF = 2
+    };
+
+    /** @brief Kind of Volume
+        Values can be TSDF (single volume) or HASHTSDF (hashtable of volume units)
+    */
+    CV_PROP_RW int kind;
```

Comment:
> int

Why int? You have already defined enum type above.

Reply:
done